[2023-06-05 18:41:09,605], [ERROR], [Internal Server Error: /get/1/update/], [C:\Users\root\Desktop\project\venv\lib\site-packages\django\utils\log.py], [(<class 'AttributeError'>, AttributeError("'__proxy__' object has no attribute 'get'"), <traceback object at 0x00000223E9641440>)]
Traceback (most recent call last):
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\utils\deprecation.py", line 136, in __call__
    response = self.process_response(request, response)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\middleware\clickjacking.py", line 27, in process_response
    if response.get("X-Frame-Options") is not None:
AttributeError: '__proxy__' object has no attribute 'get'
[2023-06-05 18:41:09,613], [ERROR], ["GET /get/1/update/ HTTP/1.1" 500 71300], [C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\servers\basehttp.py], [None]
[2023-06-05 19:07:56,623], [ERROR], [Internal Server Error: /get/board/], [C:\Users\root\Desktop\project\venv\lib\site-packages\django\utils\log.py], [(<class 'django.template.exceptions.TemplateSyntaxError'>, TemplateSyntaxError("'STATIC' is not a registered tag library. Must be one of:\naccount\nadmin_list\nadmin_modify\nadmin_urls\ncache\ni18n\nl10n\nlog\npaginator_tag\nsocialaccount\nstatic\ntz"), <traceback object at 0x00000195FE5E1580>)]
Traceback (most recent call last):
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 1026, in find_library
    return parser.libraries[name]
KeyError: 'STATIC'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\response.py", line 92, in rendered_content
    return template.render(context, self._request)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 175, in render
    return self._render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\loader_tags.py", line 132, in render
    compiled_parent = self.get_parent(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\loader_tags.py", line 129, in get_parent
    return self.find_template(parent, context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\loader_tags.py", line 107, in find_template
    template, origin = context.template.engine.find_template(
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\engine.py", line 157, in find_template
    template = loader.get_template(name, skip=skip)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 200, in compile_nodelist
    return parser.parse()
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 513, in parse
    raise self.error(token, e)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 511, in parse
    compiled_result = compile_func(self, token)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 1088, in load
    lib = find_library(parser, name)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 1028, in find_library
    raise TemplateSyntaxError(
django.template.exceptions.TemplateSyntaxError: 'STATIC' is not a registered tag library. Must be one of:
account
admin_list
admin_modify
admin_urls
cache
i18n
l10n
log
paginator_tag
socialaccount
static
tz
[2023-06-05 19:07:56,639], [ERROR], ["GET /get/board/ HTTP/1.1" 500 180148], [C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\servers\basehttp.py], [None]
[2023-06-05 19:08:14,270], [ERROR], [Internal Server Error: /get/board/], [C:\Users\root\Desktop\project\venv\lib\site-packages\django\utils\log.py], [(<class 'django.template.exceptions.TemplateSyntaxError'>, TemplateSyntaxError("Invalid block tag on line 11: 'STATIC'. Did you forget to register or load this tag?"), <traceback object at 0x00000195FE6C7D80>)]
Traceback (most recent call last):
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 505, in parse
    compile_func = self.tags[command]
KeyError: 'STATIC'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\response.py", line 92, in rendered_content
    return template.render(context, self._request)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 175, in render
    return self._render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\loader_tags.py", line 132, in render
    compiled_parent = self.get_parent(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\loader_tags.py", line 129, in get_parent
    return self.find_template(parent, context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\loader_tags.py", line 107, in find_template
    template, origin = context.template.engine.find_template(
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\engine.py", line 157, in find_template
    template = loader.get_template(name, skip=skip)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 200, in compile_nodelist
    return parser.parse()
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 507, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 568, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 11: 'STATIC'. Did you forget to register or load this tag?
[2023-06-05 19:08:14,270], [ERROR], ["GET /get/board/ HTTP/1.1" 500 172913], [C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\servers\basehttp.py], [None]
[2023-06-05 19:33:36,114], [ERROR], [Internal Server Error: /get/board/], [C:\Users\root\Desktop\project\venv\lib\site-packages\django\utils\log.py], [(<class 'django.template.exceptions.TemplateDoesNotExist'>, TemplateDoesNotExist('create_button.html'), <traceback object at 0x000002DC9C12A0C0>)]
Traceback (most recent call last):
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 175, in render
    return self._render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\loader_tags.py", line 157, in render
    return compiled_parent._render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\loader_tags.py", line 197, in render
    template = context.template.engine.select_template(template_name)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\engine.py", line 212, in select_template
    raise TemplateDoesNotExist(", ".join(not_found))
django.template.exceptions.TemplateDoesNotExist: create_button.html

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\response.py", line 92, in rendered_content
    return template.render(context, self._request)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\backends\django.py", line 63, in render
    reraise(exc, self.backend)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\backends\django.py", line 84, in reraise
    raise new from exc
django.template.exceptions.TemplateDoesNotExist: create_button.html
[2023-06-05 19:33:36,129], [ERROR], ["GET /get/board/ HTTP/1.1" 500 155904], [C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\servers\basehttp.py], [None]
[2023-06-05 19:59:07,377], [ERROR], [Internal Server Error: /get/board/], [C:\Users\root\Desktop\project\venv\lib\site-packages\django\utils\log.py], [(<class 'django.urls.exceptions.NoReverseMatch'>, NoReverseMatch("Reverse for 'logout' not found. 'logout' is not a valid view function or pattern name."), <traceback object at 0x000002DC9C1A8840>)]
Traceback (most recent call last):
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\response.py", line 92, in rendered_content
    return template.render(context, self._request)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 175, in render
    return self._render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\loader_tags.py", line 157, in render
    return compiled_parent._render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\loader_tags.py", line 63, in render
    result = block.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\loader_tags.py", line 208, in render
    return template.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 177, in render
    return self._render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 321, in render
    return nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 471, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\urls\resolvers.py", line 828, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'logout' not found. 'logout' is not a valid view function or pattern name.
[2023-06-05 19:59:07,392], [ERROR], ["GET /get/board/ HTTP/1.1" 500 214777], [C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\servers\basehttp.py], [None]
[2023-06-05 20:02:00,617], [ERROR], [Internal Server Error: /get/feedbacks/], [C:\Users\root\Desktop\project\venv\lib\site-packages\django\utils\log.py], [(<class 'django.urls.exceptions.NoReverseMatch'>, NoReverseMatch("Reverse for 'feedbacks' not found. 'feedbacks' is not a valid view function or pattern name."), <traceback object at 0x000002DC9C1281C0>)]
Traceback (most recent call last):
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\contrib\auth\decorators.py", line 23, in _wrapper_view
    return view_func(request, *args, **kwargs)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\utils\decorators.py", line 134, in _wrapper_view
    response = view_func(request, *args, **kwargs)
  File "C:\Users\root\Desktop\project\D16\APPS\bulletin_board\views.py", line 64, in consideration
    return render(request, 'page_consideration.html', context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 175, in render
    return self._render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\loader_tags.py", line 157, in render
    return compiled_parent._render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\loader_tags.py", line 63, in render
    result = block.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 471, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\urls\resolvers.py", line 828, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'feedbacks' not found. 'feedbacks' is not a valid view function or pattern name.
[2023-06-05 20:02:00,630], [ERROR], ["GET /get/feedbacks/ HTTP/1.1" 500 179139], [C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\servers\basehttp.py], [None]
[2023-06-05 22:17:54,720], [ERROR], [Forbidden (Permission denied): /get/1/], [C:\Users\root\Desktop\project\venv\lib\site-packages\django\utils\log.py], [(<class 'django.core.exceptions.PermissionDenied'>, PermissionDenied(''), <traceback object at 0x000002893BF82780>)]
Traceback (most recent call last):
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\contrib\auth\mixins.py", line 72, in dispatch
    return self.handle_no_permission()
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\contrib\auth\mixins.py", line 48, in handle_no_permission
    raise PermissionDenied(self.get_permission_denied_message())
django.core.exceptions.PermissionDenied
[2023-06-05 22:17:54,720], [ERROR], ["GET /get/1/ HTTP/1.1" 500 137250], [C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\servers\basehttp.py], [None]
[2023-06-05 22:52:12,744], [ERROR], [Internal Server Error: /accounts/login/], [C:\Users\root\Desktop\project\venv\lib\site-packages\django\utils\log.py], [(<class 'django.urls.exceptions.NoReverseMatch'>, NoReverseMatch("Reverse for 'account_singup' not found. 'account_singup' is not a valid view function or pattern name."), <traceback object at 0x00000241D2FB6A00>)]
Traceback (most recent call last):
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\response.py", line 92, in rendered_content
    return template.render(context, self._request)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 175, in render
    return self._render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\loader_tags.py", line 157, in render
    return compiled_parent._render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\loader_tags.py", line 63, in render
    result = block.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 471, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\urls\resolvers.py", line 828, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'account_singup' not found. 'account_singup' is not a valid view function or pattern name.
[2023-06-05 22:52:12,744], [ERROR], ["GET /accounts/login/ HTTP/1.1" 500 165544], [C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\servers\basehttp.py], [None]
[2023-06-05 23:24:45,818], [ERROR], [Internal Server Error: /profile/], [C:\Users\root\Desktop\project\venv\lib\site-packages\django\utils\log.py], [(<class 'django.urls.exceptions.NoReverseMatch'>, NoReverseMatch("Reverse for '' not found. '' is not a valid view function or pattern name."), <traceback object at 0x000001F5A327CB80>)]
Traceback (most recent call last):
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\response.py", line 92, in rendered_content
    return template.render(context, self._request)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 175, in render
    return self._render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\loader_tags.py", line 157, in render
    return compiled_parent._render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\loader_tags.py", line 63, in render
    result = block.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 471, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\urls\resolvers.py", line 828, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for '' not found. '' is not a valid view function or pattern name.
[2023-06-05 23:24:45,833], [ERROR], ["GET /profile/ HTTP/1.1" 500 168859], [C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\servers\basehttp.py], [None]
[2023-06-05 23:25:30,845], [ERROR], [Internal Server Error: /profile/], [C:\Users\root\Desktop\project\venv\lib\site-packages\django\utils\log.py], [(<class 'django.urls.exceptions.NoReverseMatch'>, NoReverseMatch("Reverse for 'update' with keyword arguments '{'pk': ''}' not found. 1 pattern(s) tried: ['(?P<pk>[0-9]+)/update/\\\\Z']"), <traceback object at 0x000001F5A355DF00>)]
Traceback (most recent call last):
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\response.py", line 92, in rendered_content
    return template.render(context, self._request)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 175, in render
    return self._render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\loader_tags.py", line 157, in render
    return compiled_parent._render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\loader_tags.py", line 63, in render
    result = block.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 321, in render
    return nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 238, in render
    nodelist.append(node.render_annotated(context))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 321, in render
    return nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 471, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\urls\resolvers.py", line 828, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'update' with keyword arguments '{'pk': ''}' not found. 1 pattern(s) tried: ['(?P<pk>[0-9]+)/update/\\Z']
[2023-06-05 23:25:30,861], [ERROR], ["GET /profile/ HTTP/1.1" 500 222156], [C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\servers\basehttp.py], [None]
[2023-06-05 23:25:50,559], [ERROR], [Internal Server Error: /profile/], [C:\Users\root\Desktop\project\venv\lib\site-packages\django\utils\log.py], [(<class 'django.urls.exceptions.NoReverseMatch'>, NoReverseMatch("Reverse for 'update' with keyword arguments '{'pk': ''}' not found. 1 pattern(s) tried: ['(?P<pk>[0-9]+)/update/\\\\Z']"), <traceback object at 0x000001F5A34CFE00>)]
Traceback (most recent call last):
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\response.py", line 92, in rendered_content
    return template.render(context, self._request)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 175, in render
    return self._render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\loader_tags.py", line 157, in render
    return compiled_parent._render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\loader_tags.py", line 63, in render
    result = block.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 321, in render
    return nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 238, in render
    nodelist.append(node.render_annotated(context))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 321, in render
    return nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 471, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\urls\resolvers.py", line 828, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'update' with keyword arguments '{'pk': ''}' not found. 1 pattern(s) tried: ['(?P<pk>[0-9]+)/update/\\Z']
[2023-06-05 23:25:50,567], [ERROR], ["GET /profile/ HTTP/1.1" 500 222293], [C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\servers\basehttp.py], [None]
[2023-06-05 23:30:01,914], [ERROR], [Internal Server Error: /profile/], [C:\Users\root\Desktop\project\venv\lib\site-packages\django\utils\log.py], [(<class 'django.urls.exceptions.NoReverseMatch'>, NoReverseMatch("Reverse for 'delete' with arguments '('',)' not found. 1 pattern(s) tried: ['(?P<pk>[0-9]+)/delete/\\\\Z']"), <traceback object at 0x000001F5A36F0F00>)]
Traceback (most recent call last):
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\response.py", line 92, in rendered_content
    return template.render(context, self._request)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 175, in render
    return self._render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\loader_tags.py", line 157, in render
    return compiled_parent._render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\loader_tags.py", line 63, in render
    result = block.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 321, in render
    return nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 238, in render
    nodelist.append(node.render_annotated(context))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 321, in render
    return nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 471, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\urls\resolvers.py", line 828, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'delete' with arguments '('',)' not found. 1 pattern(s) tried: ['(?P<pk>[0-9]+)/delete/\\Z']
[2023-06-05 23:30:01,930], [ERROR], ["GET /profile/ HTTP/1.1" 500 222073], [C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\servers\basehttp.py], [None]
[2023-06-05 23:30:16,128], [ERROR], [Internal Server Error: /profile/], [C:\Users\root\Desktop\project\venv\lib\site-packages\django\utils\log.py], [(<class 'django.urls.exceptions.NoReverseMatch'>, NoReverseMatch("Reverse for 'delete' with arguments '('',)' not found. 1 pattern(s) tried: ['(?P<pk>[0-9]+)/delete/\\\\Z']"), <traceback object at 0x000001F5A338C1C0>)]
Traceback (most recent call last):
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\response.py", line 92, in rendered_content
    return template.render(context, self._request)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 175, in render
    return self._render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\loader_tags.py", line 157, in render
    return compiled_parent._render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\loader_tags.py", line 63, in render
    result = block.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 321, in render
    return nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 238, in render
    nodelist.append(node.render_annotated(context))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 321, in render
    return nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 471, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\urls\resolvers.py", line 828, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'delete' with arguments '('',)' not found. 1 pattern(s) tried: ['(?P<pk>[0-9]+)/delete/\\Z']
[2023-06-05 23:30:16,136], [ERROR], ["GET /profile/ HTTP/1.1" 500 222210], [C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\servers\basehttp.py], [None]
[2023-06-05 23:30:24,131], [ERROR], [Internal Server Error: /profile/], [C:\Users\root\Desktop\project\venv\lib\site-packages\django\utils\log.py], [(<class 'django.urls.exceptions.NoReverseMatch'>, NoReverseMatch("Reverse for 'delete' with keyword arguments '{'pk': ''}' not found. 1 pattern(s) tried: ['(?P<pk>[0-9]+)/delete/\\\\Z']"), <traceback object at 0x000001F5A3753280>)]
Traceback (most recent call last):
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\response.py", line 92, in rendered_content
    return template.render(context, self._request)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 175, in render
    return self._render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\loader_tags.py", line 157, in render
    return compiled_parent._render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\loader_tags.py", line 63, in render
    result = block.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 321, in render
    return nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 238, in render
    nodelist.append(node.render_annotated(context))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 321, in render
    return nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 471, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\urls\resolvers.py", line 828, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'delete' with keyword arguments '{'pk': ''}' not found. 1 pattern(s) tried: ['(?P<pk>[0-9]+)/delete/\\Z']
[2023-06-05 23:30:24,139], [ERROR], ["GET /profile/ HTTP/1.1" 500 222686], [C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\servers\basehttp.py], [None]
[2023-06-05 23:30:32,031], [ERROR], [Internal Server Error: /profile/], [C:\Users\root\Desktop\project\venv\lib\site-packages\django\utils\log.py], [(<class 'django.urls.exceptions.NoReverseMatch'>, NoReverseMatch("Reverse for 'delete' with keyword arguments '{'pk': ''}' not found. 1 pattern(s) tried: ['(?P<pk>[0-9]+)/delete/\\\\Z']"), <traceback object at 0x000001F5A336A000>)]
Traceback (most recent call last):
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\response.py", line 92, in rendered_content
    return template.render(context, self._request)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 175, in render
    return self._render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\loader_tags.py", line 157, in render
    return compiled_parent._render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\loader_tags.py", line 63, in render
    result = block.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 321, in render
    return nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 238, in render
    nodelist.append(node.render_annotated(context))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 321, in render
    return nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 471, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\urls\resolvers.py", line 828, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'delete' with keyword arguments '{'pk': ''}' not found. 1 pattern(s) tried: ['(?P<pk>[0-9]+)/delete/\\Z']
[2023-06-05 23:30:32,039], [ERROR], ["GET /profile/ HTTP/1.1" 500 222686], [C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\servers\basehttp.py], [None]
[2023-06-05 23:30:41,361], [ERROR], [Internal Server Error: /profile/], [C:\Users\root\Desktop\project\venv\lib\site-packages\django\utils\log.py], [(<class 'django.urls.exceptions.NoReverseMatch'>, NoReverseMatch("Reverse for 'delete' with arguments '('',)' not found. 1 pattern(s) tried: ['(?P<pk>[0-9]+)/delete/\\\\Z']"), <traceback object at 0x000001F5A3630F40>)]
Traceback (most recent call last):
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\response.py", line 92, in rendered_content
    return template.render(context, self._request)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 175, in render
    return self._render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\loader_tags.py", line 157, in render
    return compiled_parent._render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\loader_tags.py", line 63, in render
    result = block.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 321, in render
    return nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 238, in render
    nodelist.append(node.render_annotated(context))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 321, in render
    return nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 471, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\urls\resolvers.py", line 828, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'delete' with arguments '('',)' not found. 1 pattern(s) tried: ['(?P<pk>[0-9]+)/delete/\\Z']
[2023-06-05 23:30:41,369], [ERROR], ["GET /profile/ HTTP/1.1" 500 222210], [C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\servers\basehttp.py], [None]
[2023-06-05 23:31:16,066], [ERROR], [Internal Server Error: /profile/], [C:\Users\root\Desktop\project\venv\lib\site-packages\django\utils\log.py], [(<class 'django.urls.exceptions.NoReverseMatch'>, NoReverseMatch("Reverse for 'delete' with arguments '('',)' not found. 1 pattern(s) tried: ['(?P<pk>[0-9]+)/delete/\\\\Z']"), <traceback object at 0x000001FC2C79EBC0>)]
Traceback (most recent call last):
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\response.py", line 92, in rendered_content
    return template.render(context, self._request)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 175, in render
    return self._render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\loader_tags.py", line 157, in render
    return compiled_parent._render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\loader_tags.py", line 63, in render
    result = block.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 321, in render
    return nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 238, in render
    nodelist.append(node.render_annotated(context))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 321, in render
    return nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 471, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\urls\resolvers.py", line 828, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'delete' with arguments '('',)' not found. 1 pattern(s) tried: ['(?P<pk>[0-9]+)/delete/\\Z']
[2023-06-05 23:31:16,066], [ERROR], ["GET /profile/ HTTP/1.1" 500 222210], [C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\servers\basehttp.py], [None]
[2023-06-05 23:31:40,148], [ERROR], [Internal Server Error: /profile/], [C:\Users\root\Desktop\project\venv\lib\site-packages\django\utils\log.py], [(<class 'django.urls.exceptions.NoReverseMatch'>, NoReverseMatch("Reverse for 'delete' with keyword arguments '{'pk': ''}' not found. 1 pattern(s) tried: ['(?P<pk>[0-9]+)/delete/\\\\Z']"), <traceback object at 0x000001FC2C62FA40>)]
Traceback (most recent call last):
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\response.py", line 92, in rendered_content
    return template.render(context, self._request)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 175, in render
    return self._render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\loader_tags.py", line 157, in render
    return compiled_parent._render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\loader_tags.py", line 63, in render
    result = block.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 321, in render
    return nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 238, in render
    nodelist.append(node.render_annotated(context))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 321, in render
    return nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 471, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\urls\resolvers.py", line 828, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'delete' with keyword arguments '{'pk': ''}' not found. 1 pattern(s) tried: ['(?P<pk>[0-9]+)/delete/\\Z']
[2023-06-05 23:31:40,164], [ERROR], ["GET /profile/ HTTP/1.1" 500 222686], [C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\servers\basehttp.py], [None]
[2023-06-05 23:32:58,883], [ERROR], [Internal Server Error: /profile/], [C:\Users\root\Desktop\project\venv\lib\site-packages\django\utils\log.py], [(<class 'django.urls.exceptions.NoReverseMatch'>, NoReverseMatch("Reverse for 'delete' with arguments '('',)' not found. 1 pattern(s) tried: ['(?P<pk>[0-9]+)/delete/\\\\Z']"), <traceback object at 0x000001FC2C54EBC0>)]
Traceback (most recent call last):
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\response.py", line 92, in rendered_content
    return template.render(context, self._request)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 175, in render
    return self._render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\loader_tags.py", line 157, in render
    return compiled_parent._render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\loader_tags.py", line 63, in render
    result = block.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 321, in render
    return nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 238, in render
    nodelist.append(node.render_annotated(context))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 321, in render
    return nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 471, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\urls\resolvers.py", line 828, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'delete' with arguments '('',)' not found. 1 pattern(s) tried: ['(?P<pk>[0-9]+)/delete/\\Z']
[2023-06-05 23:32:58,896], [ERROR], ["GET /profile/ HTTP/1.1" 500 222210], [C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\servers\basehttp.py], [None]
[2023-06-05 23:33:57,503], [ERROR], [Internal Server Error: /profile/], [C:\Users\root\Desktop\project\venv\lib\site-packages\django\utils\log.py], [(<class 'django.urls.exceptions.NoReverseMatch'>, NoReverseMatch("Reverse for 'delete' with arguments '('',)' not found. 1 pattern(s) tried: ['(?P<pk>[0-9]+)/delete/\\\\Z']"), <traceback object at 0x000001FC2C570240>)]
Traceback (most recent call last):
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\response.py", line 92, in rendered_content
    return template.render(context, self._request)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 175, in render
    return self._render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\loader_tags.py", line 157, in render
    return compiled_parent._render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\loader_tags.py", line 63, in render
    result = block.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 321, in render
    return nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 238, in render
    nodelist.append(node.render_annotated(context))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 321, in render
    return nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 471, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\urls\resolvers.py", line 828, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'delete' with arguments '('',)' not found. 1 pattern(s) tried: ['(?P<pk>[0-9]+)/delete/\\Z']
[2023-06-05 23:33:57,518], [ERROR], ["GET /profile/ HTTP/1.1" 500 222620], [C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\servers\basehttp.py], [None]
[2023-06-05 23:34:03,820], [ERROR], [Internal Server Error: /profile/], [C:\Users\root\Desktop\project\venv\lib\site-packages\django\utils\log.py], [(<class 'django.urls.exceptions.NoReverseMatch'>, NoReverseMatch("Reverse for 'delete' with arguments '('',)' not found. 1 pattern(s) tried: ['(?P<pk>[0-9]+)/delete/\\\\Z']"), <traceback object at 0x000001FC2C573700>)]
Traceback (most recent call last):
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\response.py", line 92, in rendered_content
    return template.render(context, self._request)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 175, in render
    return self._render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\loader_tags.py", line 157, in render
    return compiled_parent._render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\loader_tags.py", line 63, in render
    result = block.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 321, in render
    return nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 238, in render
    nodelist.append(node.render_annotated(context))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 321, in render
    return nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 471, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\urls\resolvers.py", line 828, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'delete' with arguments '('',)' not found. 1 pattern(s) tried: ['(?P<pk>[0-9]+)/delete/\\Z']
[2023-06-05 23:34:03,835], [ERROR], ["GET /profile/ HTTP/1.1" 500 222658], [C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\servers\basehttp.py], [None]
[2023-06-05 23:34:16,464], [ERROR], [Internal Server Error: /profile/], [C:\Users\root\Desktop\project\venv\lib\site-packages\django\utils\log.py], [(<class 'django.urls.exceptions.NoReverseMatch'>, NoReverseMatch("Reverse for 'delete' with arguments '('',)' not found. 1 pattern(s) tried: ['(?P<pk>[0-9]+)/delete/\\\\Z']"), <traceback object at 0x000001FC2C59DE40>)]
Traceback (most recent call last):
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\response.py", line 92, in rendered_content
    return template.render(context, self._request)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 175, in render
    return self._render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\loader_tags.py", line 157, in render
    return compiled_parent._render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\loader_tags.py", line 63, in render
    result = block.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 321, in render
    return nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 238, in render
    nodelist.append(node.render_annotated(context))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 321, in render
    return nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 471, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\urls\resolvers.py", line 828, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'delete' with arguments '('',)' not found. 1 pattern(s) tried: ['(?P<pk>[0-9]+)/delete/\\Z']
[2023-06-05 23:34:16,479], [ERROR], ["GET /profile/ HTTP/1.1" 500 222224], [C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\servers\basehttp.py], [None]
[2023-06-05 23:34:34,809], [ERROR], [Internal Server Error: /profile/], [C:\Users\root\Desktop\project\venv\lib\site-packages\django\utils\log.py], [(<class 'django.urls.exceptions.NoReverseMatch'>, NoReverseMatch("Reverse for 'delete' with arguments '('',)' not found. 1 pattern(s) tried: ['(?P<pk>[0-9]+)/delete/\\\\Z']"), <traceback object at 0x000001FC2CA4F000>)]
Traceback (most recent call last):
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\response.py", line 92, in rendered_content
    return template.render(context, self._request)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 175, in render
    return self._render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\loader_tags.py", line 157, in render
    return compiled_parent._render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\loader_tags.py", line 63, in render
    result = block.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 321, in render
    return nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 238, in render
    nodelist.append(node.render_annotated(context))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 321, in render
    return nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 471, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\urls\resolvers.py", line 828, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'delete' with arguments '('',)' not found. 1 pattern(s) tried: ['(?P<pk>[0-9]+)/delete/\\Z']
[2023-06-05 23:34:34,824], [ERROR], ["GET /profile/ HTTP/1.1" 500 222210], [C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\servers\basehttp.py], [None]
[2023-06-05 23:34:37,613], [ERROR], [Internal Server Error: /profile/], [C:\Users\root\Desktop\project\venv\lib\site-packages\django\utils\log.py], [(<class 'django.urls.exceptions.NoReverseMatch'>, NoReverseMatch("Reverse for 'delete' with arguments '('',)' not found. 1 pattern(s) tried: ['(?P<pk>[0-9]+)/delete/\\\\Z']"), <traceback object at 0x000001FC2CA4D700>)]
Traceback (most recent call last):
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\response.py", line 92, in rendered_content
    return template.render(context, self._request)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 175, in render
    return self._render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\loader_tags.py", line 157, in render
    return compiled_parent._render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\loader_tags.py", line 63, in render
    result = block.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 321, in render
    return nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 238, in render
    nodelist.append(node.render_annotated(context))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 321, in render
    return nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 471, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\urls\resolvers.py", line 828, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'delete' with arguments '('',)' not found. 1 pattern(s) tried: ['(?P<pk>[0-9]+)/delete/\\Z']
[2023-06-05 23:34:37,629], [ERROR], ["GET /profile/ HTTP/1.1" 500 222210], [C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\servers\basehttp.py], [None]
[2023-06-05 23:34:55,511], [ERROR], [Internal Server Error: /profile/], [C:\Users\root\Desktop\project\venv\lib\site-packages\django\utils\log.py], [(<class 'django.urls.exceptions.NoReverseMatch'>, NoReverseMatch("Reverse for 'delete' with arguments '('',)' not found. 1 pattern(s) tried: ['(?P<pk>[0-9]+)/delete/\\\\Z']"), <traceback object at 0x000001FC2C61EBC0>)]
Traceback (most recent call last):
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\response.py", line 92, in rendered_content
    return template.render(context, self._request)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 175, in render
    return self._render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\loader_tags.py", line 157, in render
    return compiled_parent._render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\loader_tags.py", line 63, in render
    result = block.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 321, in render
    return nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 238, in render
    nodelist.append(node.render_annotated(context))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 321, in render
    return nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 471, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\urls\resolvers.py", line 828, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'delete' with arguments '('',)' not found. 1 pattern(s) tried: ['(?P<pk>[0-9]+)/delete/\\Z']
[2023-06-05 23:34:55,527], [ERROR], ["GET /profile/ HTTP/1.1" 500 222210], [C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\servers\basehttp.py], [None]
[2023-06-05 23:35:08,319], [ERROR], [Internal Server Error: /profile/], [C:\Users\root\Desktop\project\venv\lib\site-packages\django\utils\log.py], [(<class 'django.urls.exceptions.NoReverseMatch'>, NoReverseMatch("Reverse for 'delete' with keyword arguments '{'pk': ''}' not found. 1 pattern(s) tried: ['(?P<pk>[0-9]+)/delete/\\\\Z']"), <traceback object at 0x000001FC2C5682C0>)]
Traceback (most recent call last):
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\response.py", line 92, in rendered_content
    return template.render(context, self._request)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 175, in render
    return self._render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\loader_tags.py", line 157, in render
    return compiled_parent._render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\loader_tags.py", line 63, in render
    result = block.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 321, in render
    return nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 238, in render
    nodelist.append(node.render_annotated(context))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 321, in render
    return nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 471, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\urls\resolvers.py", line 828, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'delete' with keyword arguments '{'pk': ''}' not found. 1 pattern(s) tried: ['(?P<pk>[0-9]+)/delete/\\Z']
[2023-06-05 23:35:08,335], [ERROR], ["GET /profile/ HTTP/1.1" 500 222686], [C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\servers\basehttp.py], [None]
[2023-06-05 23:35:22,859], [ERROR], [Internal Server Error: /profile/], [C:\Users\root\Desktop\project\venv\lib\site-packages\django\utils\log.py], [(<class 'django.urls.exceptions.NoReverseMatch'>, NoReverseMatch("Reverse for 'delete' with keyword arguments '{'pk': ''}' not found. 1 pattern(s) tried: ['(?P<pk>[0-9]+)/delete/\\\\Z']"), <traceback object at 0x000001FC2C5D5440>)]
Traceback (most recent call last):
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\response.py", line 92, in rendered_content
    return template.render(context, self._request)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 175, in render
    return self._render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\loader_tags.py", line 157, in render
    return compiled_parent._render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\loader_tags.py", line 63, in render
    result = block.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 321, in render
    return nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 238, in render
    nodelist.append(node.render_annotated(context))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 321, in render
    return nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 471, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\urls\resolvers.py", line 828, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'delete' with keyword arguments '{'pk': ''}' not found. 1 pattern(s) tried: ['(?P<pk>[0-9]+)/delete/\\Z']
[2023-06-05 23:35:22,859], [ERROR], ["GET /profile/ HTTP/1.1" 500 222686], [C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\servers\basehttp.py], [None]
[2023-06-05 23:35:35,890], [ERROR], [Internal Server Error: /profile/], [C:\Users\root\Desktop\project\venv\lib\site-packages\django\utils\log.py], [(<class 'django.urls.exceptions.NoReverseMatch'>, NoReverseMatch("Reverse for 'delete' with no arguments not found. 1 pattern(s) tried: ['(?P<pk>[0-9]+)/delete/\\\\Z']"), <traceback object at 0x000001FC2C3D4E00>)]
Traceback (most recent call last):
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\response.py", line 92, in rendered_content
    return template.render(context, self._request)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 175, in render
    return self._render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\loader_tags.py", line 157, in render
    return compiled_parent._render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\loader_tags.py", line 63, in render
    result = block.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 321, in render
    return nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 238, in render
    nodelist.append(node.render_annotated(context))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 321, in render
    return nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 471, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\urls\resolvers.py", line 828, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'delete' with no arguments not found. 1 pattern(s) tried: ['(?P<pk>[0-9]+)/delete/\\Z']
[2023-06-05 23:35:35,906], [ERROR], ["GET /profile/ HTTP/1.1" 500 220679], [C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\servers\basehttp.py], [None]
[2023-06-05 23:35:50,051], [ERROR], [Internal Server Error: /profile/], [C:\Users\root\Desktop\project\venv\lib\site-packages\django\utils\log.py], [(<class 'django.urls.exceptions.NoReverseMatch'>, NoReverseMatch("Reverse for 'delete' with keyword arguments '{'pk': ''}' not found. 1 pattern(s) tried: ['(?P<pk>[0-9]+)/delete/\\\\Z']"), <traceback object at 0x000001FC2C5C1680>)]
Traceback (most recent call last):
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\response.py", line 92, in rendered_content
    return template.render(context, self._request)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 175, in render
    return self._render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\loader_tags.py", line 157, in render
    return compiled_parent._render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\loader_tags.py", line 63, in render
    result = block.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 321, in render
    return nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 238, in render
    nodelist.append(node.render_annotated(context))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 321, in render
    return nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 471, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\urls\resolvers.py", line 828, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'delete' with keyword arguments '{'pk': ''}' not found. 1 pattern(s) tried: ['(?P<pk>[0-9]+)/delete/\\Z']
[2023-06-05 23:35:50,066], [ERROR], ["GET /profile/ HTTP/1.1" 500 222686], [C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\servers\basehttp.py], [None]
[2023-06-05 23:36:45,866], [ERROR], [Internal Server Error: /profile/], [C:\Users\root\Desktop\project\venv\lib\site-packages\django\utils\log.py], [(<class 'django.urls.exceptions.NoReverseMatch'>, NoReverseMatch("Reverse for 'delete' with arguments '('',)' not found. 1 pattern(s) tried: ['(?P<pk>[0-9]+)/delete/\\\\Z']"), <traceback object at 0x000001FC2C5B2780>)]
Traceback (most recent call last):
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\response.py", line 92, in rendered_content
    return template.render(context, self._request)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 175, in render
    return self._render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\loader_tags.py", line 157, in render
    return compiled_parent._render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\loader_tags.py", line 63, in render
    result = block.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 321, in render
    return nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 238, in render
    nodelist.append(node.render_annotated(context))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 321, in render
    return nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 471, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\urls\resolvers.py", line 828, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'delete' with arguments '('',)' not found. 1 pattern(s) tried: ['(?P<pk>[0-9]+)/delete/\\Z']
[2023-06-05 23:36:45,890], [ERROR], ["GET /profile/ HTTP/1.1" 500 222319], [C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\servers\basehttp.py], [None]
[2023-06-05 23:37:25,106], [ERROR], [Internal Server Error: /profile/], [C:\Users\root\Desktop\project\venv\lib\site-packages\django\utils\log.py], [(<class 'django.urls.exceptions.NoReverseMatch'>, NoReverseMatch("Reverse for 'delete' with arguments '('',)' not found. 1 pattern(s) tried: ['(?P<pk>[0-9]+)/delete/\\\\Z']"), <traceback object at 0x000001FC2C5C5580>)]
Traceback (most recent call last):
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\response.py", line 92, in rendered_content
    return template.render(context, self._request)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 175, in render
    return self._render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\loader_tags.py", line 157, in render
    return compiled_parent._render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\loader_tags.py", line 63, in render
    result = block.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 321, in render
    return nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 238, in render
    nodelist.append(node.render_annotated(context))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 321, in render
    return nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 471, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\urls\resolvers.py", line 828, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'delete' with arguments '('',)' not found. 1 pattern(s) tried: ['(?P<pk>[0-9]+)/delete/\\Z']
[2023-06-05 23:37:25,120], [ERROR], ["GET /profile/ HTTP/1.1" 500 222208], [C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\servers\basehttp.py], [None]
[2023-06-05 23:37:39,726], [ERROR], [Internal Server Error: /profile/], [C:\Users\root\Desktop\project\venv\lib\site-packages\django\utils\log.py], [(<class 'django.urls.exceptions.NoReverseMatch'>, NoReverseMatch("Reverse for '' not found. '' is not a valid view function or pattern name."), <traceback object at 0x000001FC2C8C6380>)]
Traceback (most recent call last):
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\response.py", line 92, in rendered_content
    return template.render(context, self._request)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 175, in render
    return self._render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\loader_tags.py", line 157, in render
    return compiled_parent._render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\loader_tags.py", line 63, in render
    result = block.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 321, in render
    return nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 238, in render
    nodelist.append(node.render_annotated(context))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 321, in render
    return nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 471, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\urls\resolvers.py", line 828, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for '' not found. '' is not a valid view function or pattern name.
[2023-06-05 23:37:39,742], [ERROR], ["GET /profile/ HTTP/1.1" 500 219121], [C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\servers\basehttp.py], [None]
[2023-06-05 23:39:18,564], [ERROR], [Internal Server Error: /profile/], [C:\Users\root\Desktop\project\venv\lib\site-packages\django\utils\log.py], [(<class 'django.urls.exceptions.NoReverseMatch'>, NoReverseMatch("Reverse for 'delete' with arguments '('',)' not found. 1 pattern(s) tried: ['(?P<pk>[0-9]+)/delete/\\\\Z']"), <traceback object at 0x000001FC2C66F000>)]
Traceback (most recent call last):
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\response.py", line 92, in rendered_content
    return template.render(context, self._request)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 175, in render
    return self._render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\loader_tags.py", line 157, in render
    return compiled_parent._render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\loader_tags.py", line 63, in render
    result = block.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 321, in render
    return nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 238, in render
    nodelist.append(node.render_annotated(context))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 321, in render
    return nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 471, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\urls\resolvers.py", line 828, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'delete' with arguments '('',)' not found. 1 pattern(s) tried: ['(?P<pk>[0-9]+)/delete/\\Z']
[2023-06-05 23:39:18,580], [ERROR], ["GET /profile/ HTTP/1.1" 500 222208], [C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\servers\basehttp.py], [None]
[2023-06-05 23:48:34,523], [ERROR], [Internal Server Error: /profile/], [C:\Users\root\Desktop\project\venv\lib\site-packages\django\utils\log.py], [(<class 'django.core.exceptions.ImproperlyConfigured'>, ImproperlyConfigured('Profile is missing a QuerySet. Define Profile.model, Profile.queryset, or override Profile.get_queryset().'), <traceback object at 0x000001F7A3D2E000>)]
Traceback (most recent call last):
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\views\generic\base.py", line 104, in view
    return self.dispatch(request, *args, **kwargs)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\views\generic\base.py", line 143, in dispatch
    return handler(request, *args, **kwargs)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\views\generic\list.py", line 154, in get
    self.object_list = self.get_queryset()
  File "C:\Users\root\Desktop\project\D16\APPS\bulletin_board\views.py", line 22, in get_queryset
    queryset = super().get_queryset()
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\views\generic\list.py", line 36, in get_queryset
    raise ImproperlyConfigured(
django.core.exceptions.ImproperlyConfigured: Profile is missing a QuerySet. Define Profile.model, Profile.queryset, or override Profile.get_queryset().
[2023-06-05 23:48:34,531], [ERROR], ["GET /profile/?header__istartswith=s&category=&added_after= HTTP/1.1" 500 90298], [C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\servers\basehttp.py], [None]
[2023-06-05 23:54:20,878], [ERROR], [Internal Server Error: /profile/], [C:\Users\root\Desktop\project\venv\lib\site-packages\django\utils\log.py], [(<class 'django.urls.exceptions.NoReverseMatch'>, NoReverseMatch("Reverse for 'delete' with arguments '('',)' not found. 1 pattern(s) tried: ['(?P<pk>[0-9]+)/delete/\\\\Z']"), <traceback object at 0x000002963A5A85C0>)]
Traceback (most recent call last):
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\handlers\base.py", line 220, in _get_response
    response = response.render()
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\response.py", line 114, in render
    self.content = self.rendered_content
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\response.py", line 92, in rendered_content
    return template.render(context, self._request)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 175, in render
    return self._render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\loader_tags.py", line 157, in render
    return compiled_parent._render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\loader_tags.py", line 63, in render
    result = block.nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 321, in render
    return nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 238, in render
    nodelist.append(node.render_annotated(context))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 321, in render
    return nodelist.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\template\defaulttags.py", line 471, in render
    url = reverse(view_name, args=args, kwargs=kwargs, current_app=current_app)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\urls\base.py", line 88, in reverse
    return resolver._reverse_with_prefix(view, prefix, *args, **kwargs)
  File "C:\Users\root\Desktop\project\venv\lib\site-packages\django\urls\resolvers.py", line 828, in _reverse_with_prefix
    raise NoReverseMatch(msg)
django.urls.exceptions.NoReverseMatch: Reverse for 'delete' with arguments '('',)' not found. 1 pattern(s) tried: ['(?P<pk>[0-9]+)/delete/\\Z']
[2023-06-05 23:54:20,910], [ERROR], ["GET /profile/?header__istartswith=&category=1&added_after= HTTP/1.1" 500 218585], [C:\Users\root\Desktop\project\venv\lib\site-packages\django\core\servers\basehttp.py], [None]
